languages/elixir/exercises/concept/pattern-matching/lib/date_string.ex

```diff
@@ -0,0 +1,7 @@
+defmodule DateString do
+
+  # TODO: implement the from_tuple/1 function
```

Comment:
```suggestion
  do from_tuple(tuple) do
    raise "Please implement the from_tuple/2 function."
  end
```

add stubs to this module which then raise a human readable error on testing

Reply:
That should be `raise "Please implement the from_tuple/1 function."` I think

Reply:
Right